tools/native/build-native.ps1

```
@@ -0,0 +1,217 @@
+    return 0
+}
+
+function Resolve-WindowsGeneratorPlan {
+    param(
+        [string]$Configuration,
+        [string]$RecommendedGenerator,
+        [string]$VsInstancePath
+    )
+
+    if (-not [string]::IsNullOrWhiteSpace($RecommendedGenerator)) {
+        $args = @("-G", $RecommendedGenerator, "-A", "x64")
```

PSScriptAnalyzer (1.24.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)